meson.build

```diff
@@ -0,0 +1,30 @@
+project('eos-parental-controls', 'c',
+  version : '0.1.0',
+  meson_version : '>= 0.47.0',
+  license: 'LGPLv2.1+',
+  default_options : [
+    'buildtype=debugoptimized',
+    'warning_level=2',
+    'c_std=gnu11',
+  ]
+)
+
+pkgconfig = import('pkgconfig')
+
+meson_make_symlink = join_paths(meson.source_root(), 'tools', 'meson-make-symlink.sh')
+
+prefix = get_option('prefix')
+datadir = join_paths(prefix, get_option('datadir'))
+
+# FIXME: This isn’t exposed in accountsservice.pc
+accountsserviceinterfacesdir = join_paths(datadir, 'accountsservice', 'interfaces')
+
+dbus = dependency('dbus-1')
+dbusinterfacesdir = dbus.get_pkgconfig_variable('interfaces_dir',
+  define_variable: ['datadir', datadir])
```

Comment:
Huh, this is a new one to me. I wonder if this can replace a number of places where an extra option gets added to meson_options.txt to override these kinds of dirs.

Reply:
That’s the idea. It’s pretty neat. Downside: it does require you to (for example) have libsystemd-dev installed if you want to know where to install a `.service` file, even if you never actually link against libsystemd.